### ISSUE TITLE
when switching from dongle to bluetooth, mouse stops working

### ISSUE THREAD

**Issue by mhantsch** [UltimateHackingKeyboard/firmware#939]
When switching between Bluetooth and Dongle back and forth, a few things go wrong:
- The diplay continues to show `UHK Dongle` even after I have pulled the dongle an the connection is now on Bluetooth.
- The mouse stops working. Cannot move the mouse pointer or click. It doesn't matter whether using mouse layer keys, or touchpad.

Have to restart the UHK80 to get "Bluetooth" displayed again. Then the mouse also works again.

**Comment by kareltucek** [comment]
Only mouse?

It would make sense for uhk to (try to) send the reports over to the Dongle if uhk thinks it is connected, but that should affect all usb interfaces 🤨.

**Comment by mhantsch** [comment]
Yes, it was only the mouse that stopped working. It did not matter whether I tried using the touchpad or mouse layer and keys, or clicking using keys configured for mouse clicks. Normal keyboard (scancodes) would work, but all mouse functionality was gone.

**Comment by mondalaci** [comment]
I can reproduce this issue, but it's not obvious.

First, I tried to switch between the dongle and my Android phone. Once one is connected, the other can't connect because the UHK 80 doesn't advertise via BLE anymore. So if the dongle is connected, it must be disconnected, or if my phone is connected, I have to disable Bluetooth on it. Then, the switchover happens.

Switching back and forth this way didn't reproduce the issue. Then I was connected to the dongle and started up Agent, disconnected the dongle, Agent recognized the UHK 80, and then mouse actions didn't work anymore while key actions worked. Rebooting the right half fixed the issue.

I also reported the stuck dongle issue separately in UltimateHackingKeyboard/firmware#1002.
